modules/exploits/linux/http/nagios_xi_configwizards_authenticated_rce.rb

```diff
@@ -0,0 +1,195 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HTTP::NagiosXi
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Nagios XI 5.7.5 - ConfigWizards Authenticated Remote Code Exection',
+        'Description' => %q{
+          This module exploits CVE-2021-25296, CVE-2021-25297, or CVE-2021-25298,
+          OS command injection vulnerabilities in several URL parameters that enables
+          an authenticated user to perform remote code execution on Nagios XI 5.7.5 as
+          the apache user.
+
+          Valid credentials for a Nagios XI user are required. This module has
+          been successfully tested against the official NagiosXI 5.7.5 OVA.
```

Comment:
Do we know if it affects prior versions at all? The advisories only list 5.7.5 but doesn't look like anyone tried checking older versions to see if it still applies or is specific to just this version.

Reply:
I don't know right now, but after I address the rest of the comments I'll pull a few older OVAs and see if I can get an idea of whether it does or not

Reply:
Looks like this was updated and we confirmed older versions are indeed vulnerable. Marking resolved.